gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -227,16 +227,17 @@ gst_tensor_query_serversink_start (GstBaseSink * bsink)
   GstTensorQueryServerSink *sink = GST_TENSOR_QUERY_SERVERSINK (bsink);
   GstCaps *caps;
   gchar *caps_str = NULL;
+  gboolean ret = FALSE;
```

Comment:
Well.. this variable seems not to be changed in runtime. Any merit to declare it?